README.md

```diff
@@ -63,18 +62,18 @@ Prepare a samplesheet with input data:
 
 ```
 project,sample,file,family_id,paternal_id,maternal_id,sex,phenotype
-testrun,HG002,/path/to/HG002.fastq.gz,FAM1,HG003,HG004,1,2
-testrun,HG005,/path/to/HG005.bam,FAM1,HG003,HG004,2,1
+NIST,HG002,/path/to/HG002.fastq.gz,FAM1,HG003,HG004,1,2
+NIST,HG005,/path/to/HG005.bam,FAM1,HG003,HG004,2,1
 ```
 
-Now, you can run the pipeline using:
+Supply a reference genome with `--fasta` and choose a matching `--preset` for your data (`revio`, `pacbio`, `ONT_R10`). Now, you can run the pipeline, e.g:
 
 ```bash
 nextflow run genomic-medicine-sweden/nallo -profile YOURPROFILE \
     --input samplesheet.csv \
-    --preset <revio/pacbio/ONT_R10> \
-    --fasta <reference.fasta> \
-    --outdir <OUTDIR>
+    --preset revio \
+    --fasta  reference.fasta \
```

Comment:
```suggestion
    --fasta reference.fasta \
```

Reply:
I like formatting the options within brackets better as it highlights what the user has to provide

Reply:
I'll change it back!